build-system/tasks/e2e/amp-driver.js

```diff
@@ -98,6 +99,19 @@ const EnvironmentBehaviorMap = {
     },
   },
 
+  [AmpdocEnvironment.A4A_FIE_NO_SIGNING]: {
+    async ready(controller) {
+      return controller
+        .findElement('amp-ad > iframe')
+        .then((frame) => controller.switchToFrame(frame));
+    },
+
+    url(url) {
+      const a4aUrl = url.replace(HOST, HOST + '/a4a');
+      return `${a4aUrl}?eid=a4a-no-signing:21066325`;
```

Comment:
Curious where `21066325` comes from?

Reply:
It comes from here https://github.com/ampproject/amphtml/blob/cd1e0c158d1d46e9a298f98f62ace60c6df135d0/extensions/amp-a4a/0.1/amp-a4a.js#L184

I tried to import this object directly using `require`, but it was giving me all sorts of errors. Is there a good way to do this?

Reply:
Hm not sure! Maybe best to keep it separate though, feels weird to have a testing framework depend on source code.

Reply:
@estherkim I actually think it's OK to do this, because our intention here is to have an additional env with this particular experiment enabled. And in any way it's a temporally env, that will be removed once the experiment is launched. What do you think?

Reply:
added a comment for now. Happy to switch to import if there is a better way.

Reply:
Importing it should work if you move it out of `amp-a4a.js` to its own file. Looks like MODULE_NOMODULE_PARAMS_EXP could be moved along with NO_SIGNING_EXP.

Reply:
I still don't feel it worth the effort to split this temporary ID out to a separate file.

Reply:
Ok then, either way is fine with me 🚢